msg.go

```diff
@@ -36,6 +36,9 @@ const (
 	// not something a well written implementation should ever do, so we leave them
 	// to trip the maximum compression pointer check.
 	maxCompressionPointers = (maxDomainNameWireOctets+1)/2 - 2
+
+	// This is the maximum length of a domain name in presentation format.
+	maxDomainNamePresentationLength = 61*4 + 1 + 63*4 + 1 + 63*4 + 1 + 63*4 + 1
```

Comment:
could you explain the calculation in the comment? I can reason back with the 63 (a bit), but not reconstruct it myself.

Reply:
Ack, this does need a better explanation. I was working off mostly https://blogs.msdn.microsoft.com/oldnewthing/20120412-00/?p=7873/ because it's actually harder to figure out than it would seem.

Reply:
Done. I think that should be clear enough.